content/3_Silver/Prefix_Sums.mdx

```diff
@@ -395,6 +395,21 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```python
+def psum(a):
+	s=0
```

Comment:
I'm not very familiar with python but I think you can get rid of the s variable and just use the psum array

Reply:
Yeah- it's possible for you to just set the first element as 0 and then use the last element of the array as the `s` variable.

Reply:
Okay cool. Can you modify your program and get rid of s pls. thx!

Reply:
(if you didn't notice i'm not the author lol, just was giving my 2 cents)

Reply:
ok I can do this. But why do we want to get rid of this variable? I think this makes the code clearer. Or you want the code to be consistent with the C++ version?

Reply:
I don't think that it makes the code clearer. And when you're using names like `s`, might as well just make it concise as well.